packages/cli/src/databases/entities/ProjectRelation.ts

```diff
@@ -15,7 +15,7 @@ export class ProjectRelation extends WithTimestamps {
 	@Column()
 	role: ProjectRole;
 
-	@ManyToOne('User', 'projectRelations')
+	@ManyToOne('User', 'projectRelations', { eager: true })
```

Comment:
This made getting all tests to pass relatively easy. If we disable this there are more routes that need to be fixed manually by adding the necessary relations to the repository methods.